etabotsite/etabotapp/views.py

```diff
@@ -38,23 +38,48 @@
 else:
     logger.setLevel(logging.INFO)
 
+PROD_HOST_URL = getattr(settings, "PROD_HOST_URL", "http://localhost:8000")
+
+
 def fetch_token(name, request):
+    """Authlib support function."""
+    OAUTH1_SERVICES = []
     if name in OAUTH1_SERVICES:
         model = OAuth1Token
     else:
         model = OAuth2Token
-
+    logging.info('authlib fetch_token searching for token in model {}'.format(
+        model))
     token = model.find(
         name=name,
         owner=request.user
     )
     return token.to_token()
 
-oauth = OAuth(fetch_token=fetch_token)
+
+def update_token(name, token, refresh_token=None, access_token=None):
+    """Authlib support function.""" 
+    logging.info('updating token')
+    if refresh_token:
+        logging.info('searching for token by refresh_token')
+        item = OAuth2Token.find(name=name, refresh_token=refresh_token)
```

Comment:
should the `OAuthToken` class be consistent with `fetch_token`? If `fetch_token` is using `OAuth1Token`, then maybe we should use `OAuth1Token` here too?